build/config/SignToolData.json

```diff
@@ -127,6 +127,7 @@
     "e_sqlite3.dll",
     "Humanizer.dll",
     "Humanizer.resources.dll",
+    "ICSharpCode.Decompiler.dll",
```

Comment:
Are these being signed externally or are we consumed unsigned? If the latter we have a process to handle that, we just need to start some internal paperwork.

Reply:
:arrow_right: The assembly has a strong name but is not authenticode signed.